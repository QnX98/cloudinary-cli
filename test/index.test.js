require('..')
const file = require('../config/file')

describe('upload', () => {
  const data = {
    CLOUD_NAME: process.env.CLOUD_NAME,
    API_KEY: process.env.API_KEY,
    API_SECRET: process.env.API_SECRET
  }
  it('get env file', () => {
    expect(file(data)).toMatchSnapshot()
  })
})
