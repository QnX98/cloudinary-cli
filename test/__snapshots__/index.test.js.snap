// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`upload get env file 1`] = `undefined`;
